packages/harmony/src/components/layout/Box/types.ts

```diff
@@ -15,9 +15,9 @@ export type BaseBoxProps = {
   /** Padding */
   p?: SpacingOptions
   /** Padding Horizontal */
-  ph?: SpacingOptions
```

Comment:
just double checking these changes are intentional? seems like a a bit of a philosophical change to allow custom padding.

Reply:
yeah it is a bit of a change. this has been there for a bit for paddingTop etc, so just making all the padding options consistent in this case. It his helpful to negate or one-off options so youre not having to make a css-object.